charts/consul/templates/sync-catalog-deployment.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if (or (and (ne (.Values.syncCatalog.enabled | toString) "-") .Values.syncCatalog.enabled) (and (eq (.Values.syncCatalog.enabled | toString) "-") .Values.global.enabled)) }}
+{{ - $syncCatalogEnabled := (or (and (ne (.Values.syncCatalog.enabled | toString) "-") .Values.syncCatalog.enabled) (and (eq (.Values.syncCatalog.enabled | toString) "-") .Values.global.enabled)) }}
```

Comment:
thinking about this more we should have an entirely separate job to handle this rather than re-use the sync-catalog job

Reply:
Yes, right, totally agree, I think creating a brand new one makes more sense here